playbook/app/pb_kits/playbook/pb_card/docs/_card_background.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= pb_rails("title", props: { text: "Card Colors", tag: "h4", size: 4 }) %>
+
+<br>
+
+<%= pb_rails("card", props: { background: "dark", classname: "dark" })  do %>
```

Comment:
Why the added classname of dark?

Reply:
I added the classname dark to get the card in dark mode so the text would look better, I'll get with Jon or Jasper and we can figure that out.